charters/MetadataSchema/charter.md

```diff
@@ -0,0 +1,66 @@
+
+# Metadata Schema
+
+## Description
+
+The Human Cell Atlas (HCA) Metadata Schema are JSON format schema. These schema are designed to capture and provide structure for the descriptive scientific metadata associated with HCA datasets. These schemas aim to ensure [the FAIRness of the HCA data](https://www.nature.com/articles/sdata201618).
+
+## Objective
+
+1. Ensuring that HCA data are findable and the schemas are computationally queryable
+2. Ensuring that HCA metadata supports reproducibility and reusability
+3. Ensuring that HCA metadata has a consistent representation and can accurately represent the scientific description of a dataset in a computational readable manner
+4. Defining JSON schema which enable objectives 1 and 2
```

Comment:
This use of the word JSON also seems unnecessary and more generic "Defining schema which..." may be more appropriate.

Reply:
agreed, JSON is an implementation. not an objective.

Reply:
I'm a little confused about where the line is for including/excluding implementation choices in the charters. For example, the Data Store charter indicates Python bindings, AWS/GCP for cloud infrastructures, Elastic Search Percolate for query service. The Data Portal charter indicates Google Analytics for system logging, AWS for hosting the website, Dockstore for hosting portals. What is the right level of including/excluding implementation choices? I think being consistent is key here across all the charters so we present a united front to our users. @brianraymor maybe you can provide some guidance? I can think of arguments both for and against mentioning JSON here.

Reply:
I agree that implementation details should be mostly left out but I am torn on JSON as it feels like such a fundamental part of design that changing it creates significant downstream consequences and possible actually trigger rechartering 

This sounds like something we should discuss at the face to face meeting in September and have an agreement what, if any, implementation details should be in our charters

Reply:
@malloryfreeberg - I do not believe the charters should include *internal* implementation details in their scope which should be high level. I would expect specific data formats or language bindings to be detailed in the milestones - especially since there are so many dependencies on the metadata JSON format.

Now that we have multiple charters to compare, I think that it would be helpful to focus our attention on updating Data Store as an exemplar. @kozbo has kindly agreed to withdraw his charter from architecture review this week so we can perform further review.

Reply:
As I said to @TimothyTickle above and I think the conclusion we reached in this conversation the JSON aspect of the schemas are so fundamental to the schemas that changing it would need rechartering. I am leaving it in.